character-sfx.js

```diff
@@ -342,6 +359,74 @@ class CharacterSfx {
   
     };
     _handleNarutoRun();
+
+    // combo
+    const dispatchComboSoundEvent = (combo, indexOfCombo) =>{
+      this.dispatchEvent(new MessageEvent('comboSoundEmit', {

```

Comment:
This is a poor name for the event, not consistent with other events.

Reply:
https://github.com/webaverse/silsword/pull/12/commits/5d403c4595fc49fb3c7d18267cee8aaac5d50c33
https://github.com/webaverse/app/pull/3660/commits/38d71b6bd254fcd1714666dd2465a7ce21e27b9f